src/bridge/cockpithttpstream.c

```diff
@@ -641,22 +640,6 @@ on_stream_read (CockpitStream *stream,
   g_object_unref (self);
 }
 
-static void
-on_rejected_cert (CockpitStream *stream,
-                  const gchar *pem_data,
-                  gpointer user_data)
-{
-  CockpitHttpStream *self = user_data;
-  CockpitChannel *channel = user_data;
-  JsonObject *close_options = NULL; // owned by channel
-
-  if (self->state != FINISHED)
-    {
-      close_options = cockpit_channel_close_options (channel);
-      json_object_set_string_member (close_options, "rejected-certificate", pem_data);
```

Comment:
It's that attribute in the JSON protocol that you are concerned about, right? Indeed nothing uses that, it's not documented, and personally I'm more than happy to let it go to lose all this custom code.